src/main/java/org/truffleruby/core/rope/RopeConstants.java

```diff
@@ -126,4 +139,38 @@ public static LeafRope lookupUSASCII(String string) {
         }
     }
 
+    private static final LeafRope[] PADDED_NUMBERS = new LeafRope[100];
+
+    static {
```

Comment:
Style thing: a static method returning `LeafRope[]` seems a lot cleaner than a static initializer like here.
Also, there is no risk to ever have `null` in the LeafRope[] that way.

Reply:
Done.